<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>235</width>
    <height>421</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Detoks v 0.3</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/icons/zas/icons/penguin.png</normaloff>:/icons/zas/icons/penguin.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(39, 39, 39);
alternate-background-color: rgb(52, 39, 39);
color: rgb(180, 55, 55);

border-color: rgb(211, 131, 61);

font: 9pt &quot;Calibri&quot;;


margin: 3px;
border-width: 1px;
border-radius: 5px;
border-style:outset;
</string>
  </property>
  <widget class="QWidget" name="widget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>0</y>
      <width>231</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QListWidget" name="listWidget">
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <italic>true</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
font: italic 10pt &quot;Verdana&quot;;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(52, 66, 99);
color: rgb(255, 255, 255);</string>
       </property>
       <item>
        <property name="text">
         <string>Dostępny</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/status/online</normaloff>:/status/online</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Zajęty</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/status/busy</normaloff>:/status/busy</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Niewidoczny</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/status/invisible</normaloff>:/status/invisible</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Niedostępny</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/status/offline</normaloff>:/status/offline</iconset>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>235</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionModyfikuj_list_znajomych"/>
    <addaction name="actionUstawienia"/>
    <addaction name="separator"/>
    <addaction name="actionZamknij"/>
   </widget>
   <widget class="QMenu" name="menu_czno">
    <property name="title">
     <string>Wiadmomości</string>
    </property>
    <addaction name="actionEmail"/>
    <addaction name="actionNet_send"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menu_czno"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>31</width>
     <height>24</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionUstawienia"/>
   <addaction name="actionModyfikuj_list_znajomych"/>
   <addaction name="actionEmail"/>
   <addaction name="actionNet_send"/>
   <addaction name="separator"/>
   <addaction name="actionInfo"/>
  </widget>
  <action name="actionUstawienia">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/zas/icons/preferences-system.png</normaloff>:/icons/zas/icons/preferences-system.png</iconset>
   </property>
   <property name="text">
    <string>Ustawienia</string>
   </property>
  </action>
  <action name="actionModyfikuj_list_znajomych">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/zas/icons/resource-group.png</normaloff>:/icons/zas/icons/resource-group.png</iconset>
   </property>
   <property name="text">
    <string>Modyfikuj listę znajomych</string>
   </property>
   <property name="toolTip">
    <string>Modyfikuj listę znajomych</string>
   </property>
  </action>
  <action name="actionZamknij">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/zas/icons/window-close.png</normaloff>:/icons/zas/icons/window-close.png</iconset>
   </property>
   <property name="text">
    <string>Zamknij</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionEmail">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/msg</normaloff>:/icons/msg</iconset>
   </property>
   <property name="text">
    <string>Wyślij Maila</string>
   </property>
   <property name="toolTip">
    <string>wyślij Maila</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionNet_send">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/zas/icons/konversation.png</normaloff>:/icons/zas/icons/konversation.png</iconset>
   </property>
   <property name="text">
    <string>Net send</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/zas/icons/dialog-information.png</normaloff>:/icons/zas/icons/dialog-information.png</iconset>
   </property>
   <property name="text">
    <string>Info</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionWyniki">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/emots/zas/emotki/face-wink.png</normaloff>:/emots/zas/emotki/face-wink.png</iconset>
   </property>
   <property name="text">
    <string>wyniki :)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
